build.gradle

```diff
@@ -51,7 +51,9 @@ configurations {
 }
 
 dependencies {
+    pilgrim "android.arch.lifecycle:common:1.1.1"
```

Comment:
Not exactly sure why but needed to add this to build.gradle script or else starting the android debugging activity would crash saying it could't find `android.support.v7.app.AppCompatActivity` and `android.arch.lifecycle.LifecycleOwner`

Reply:
what version of the support library is used? I dont know how dependencies work here but lifecycleowner is a transitive dependency, so its possible that the aar for that wasnt included?

Reply:
Using 2.1.2 same as pilgrim, for some reason the build.gradle seems to only pull down direct dependencies of what's declared in pom file not transitive ones